jib-core/src/main/java/com/google/cloud/tools/jib/api/JibContainerBuilder.java

```diff
@@ -305,6 +309,29 @@ public JibContainerBuilder addLabel(String key, String value) {
     return this;
   }
 
+  /**
+   * Sets the format to build the container as. Use {@link ImageFormat#Docker} for Docker V2.2 or
```

Comment:
I'm unclear on the terminology, but are "container" and "image" interchangeable here?

Also, "Sets the target format of the image" might be more clear?

Reply:
Yes, most people use the terms interchangeably, though more accurately, container refers to running a process in a certain chroot, cgroup, and namespace, whereas image refers to a representation for how and what to launch as the container. However, for simplicity sake, since we are performing "containerization", we're just referring to container and image interchangeably, but I'll actually change this to `container image` to be more precise here.